talk/morelanguage/random.tex

```diff
@@ -0,0 +1,81 @@
+\subsection[a]{Random}
+%\epsdice[white]{3}
+
+\begin{frame}[fragile]
+  \frametitlecpp[11]{Pseudo-random number generators (PRNG)}
+  \begin{block}{Concept}
+    \begin{itemize}
+      \item For generating pseudo-random numbers the standard library offers:
+      \begin{itemize}
+        \item engines and engine adaptors to generate pseudo random bits
+        \item distributions to shape these into numbers
+        \item access to (potential) hardware entropy
+      \end{itemize}
+      \item Found in header \mintinline{cpp}{<random>}
+    \end{itemize}
+  \end{block}
+  \begin{cppcode}
+
+ \end{cppcode}
+\end{frame}
+
+\begin{frame}[fragile]
+  \frametitlecpp[11]{Engines}
+  \begin{block}{Engines}
+    \begin{itemize}
+      \item Generate random bits (as integers), depending on algorithm
+      \item Can be seeded via constructor or \mintinline{cpp}{seed()}
+      \item Algorithms: \mintinline{cpp}{linear_congruential_engine}, \mintinline{cpp}{mersenne_twister_engine}, \mintinline{cpp}{subtract_with_carry_engine}
+      \item Adaptors: \mintinline{cpp}{discard_block_engine}, \mintinline{cpp}{independent_bits_engine}, \mintinline{cpp}{shuffle_order_engine}
+      \item Aliases (use these): \mintinline{cpp}{minstd_rand0}, \mintinline{cpp}{minstd_rand}, \mintinline{cpp}{mt19937}, \mintinline{cpp}{mt19937_64}, \mintinline{cpp}{ranlux24}, \mintinline{cpp}{ranlux48}, \mintinline{cpp}{knuth_b}, \mintinline{cpp}{default_random_engine}
```

Comment:
maybe use small font for the names so that it fits on 2 lines less, Then you have space for a header of an example block at the bottom, with title "Typical code" or so